pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
```

Comment:
Does prometheus create a new time series if the tag of some metric changes over time ? 

Could you please give more details of the problem you foresee ? 
Would there be a very short lived time series with an empty value in the node tag? 
Is that your concern ?